docs/config.md

```diff
@@ -199,21 +199,28 @@ Obviously, you would only set one line, don't copy them all in!
 ## Editing diffs
 
 The `ui.diff-editor` setting affects the tool used for editing diffs (e.g.
-`jj split`, `jj amend -i`). The default is `meld`. The left and right
-directories to diff are passed as the first and second argument respectively.
+`jj split`, `jj amend -i`). The default is `meld`.
+
+`jj` replaces the following arguments:
+
+- `$left` and `$right` are replaced with the paths to the left and right
+  directories to diff respectively.
+
+If no interpolation parameters are found, `$left $right` is appended.
```

Comment:
(Not necessarily for this PR): Is it documented that the setting and `ui.merge-editor` can be a either list or a space-separated string?